api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -47,6 +52,11 @@
   private boolean disabled;
   private EmailVerificationStatus emailVerificationStatus;
 
+  private List<InstitutionalAffiliation> institutionalAffiliationSet =
```

Comment:
Why name the property "set" rather than "list"?

Reply:
I'm actually just going to call it institutionalAffiliations, because I think the field should be data-structure independent for name. (Also, set was a relic of an old version)